tokio-console/src/config.rs

```diff
@@ -163,11 +206,43 @@ impl Config {
     }
 }
 
+fn merge_view_options(base: &mut ViewOptions, command_line: ViewOptions) {
+    if command_line.no_colors.is_some() {
+        base.no_colors = command_line.no_colors;
+    }
+
+    if command_line.lang.is_some() {
+        base.lang = command_line.lang;
+    }
+
+    if command_line.ascii_only.is_some() {
+        base.ascii_only = command_line.ascii_only;
+    }
+
+    if command_line.truecolor.is_some() {
+        base.truecolor = command_line.truecolor;
+    }
+
+    if command_line.palette.is_some() {
+        base.palette = command_line.palette;
+    }
+
+    if command_line.toggles.color_durations.is_some() {
+        base.toggles.color_durations = command_line.toggles.color_durations;
+    }
+
+    if command_line.toggles.color_terminated.is_some() {
+        base.toggles.color_terminated = command_line.toggles.color_terminated;
+    }
+}
+
 // === impl ViewOptions ===
 
 impl ViewOptions {
     pub fn is_utf8(&self) -> bool {
-        self.lang.ends_with("UTF-8") && !self.ascii_only
+        let lang = self.lang.clone().unwrap_or_default();
```

Comment:
i don't think we need to clone `self.lang` here; we can borrow it instead:
```suggestion
        let lang = self.lang.as_ref().unwrap_or_default();
```